plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/FilesTable.java

```diff
@@ -255,6 +260,37 @@ private Object toIntegerVarcharMapBlock(Map<Integer, String> values)
             blockBuilder.closeEntry();
             return integerToVarcharMapType.getObject(blockBuilder, 0);
         }
+
+        @Nullable
+        private Block getBigintArrayBlock(List<Long> values)
+        {
+            if (values == null) {
+                return null;
+            }
+            BlockBuilder builder = BIGINT.createBlockBuilder(null, values.size());
+            values.forEach(value -> BIGINT.writeLong(builder, value));
+            return builder.build();
+        }
+
+        @Nullable
+        private Block getIntegerArrayBlock(List<Integer> values)
```

Comment:
nit: Move to before `getIntegerArrayBlock`. Integer → BigInt is the natural order.